src/scancode/extract_cli.py

```diff
@@ -159,8 +160,9 @@ def display_extract_summary():
     extract_result_with_errors = []
     unique_extract_events_with_errors = set()
     has_extract_errors = False
+    
     extractibles = extract_archives(
-        abs_location, recurse=not shallow, replace_originals=replace_originals)
+        abs_location, recurse=not shallow, replace_originals=replace_originals, ignored_extensions=ignore)
```

Comment:
Same comment as above. We probably want to keep variable naming consistent. why have the variable `name `ignored_extensions` and not just `ignore`?